pybind11_catkin/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(pybind11_catkin)
+
+find_package(catkin REQUIRED)
+
+include(ExternalProject)
+
+set(PYBIND_SRC ${CMAKE_BINARY_DIR}/pybind11_src)
+
+ExternalProject_Add(
+  pybind11
+  GIT_REPOSITORY "git@github.com:pybind/pybind11.git"
```

Comment:
This should be the HTTPS version - ``https://github.com/pybind/pybind11.git``. Otherwise will fail if you don't have a private key registered with GitHub on that machine (e.g. build server)